schema/brain/queries/reads.py

```diff
@@ -76,6 +76,12 @@ def get_plugin_command(plugin_name, command_name, conn=None):
     return command
 
 
+@wrap_rethink_errors
+@wrap_connection
```

Comment:
consider adding the new reconnect wrapper to all of these

Reply:
It looks like the connect wrapper is already wrapped with the reconnect wrapper